lib/apis/archives.js

```diff
@@ -163,6 +163,16 @@ module.exports = class ArchivesAPI {
     res.status(200).end()
   }
 
+  async report (req, res) {
+    // validate session
+    if (!res.locals.session) throw new UnauthorizedError()
+    if (!res.locals.session.scopes.includes('user')) throw new ForbiddenError()
+
+    req.checkBody('key').isDatHash()
+    ;(await req.getValidationResult()).throw()
+    var { key } = req.body
```

Comment:
Incomplete definition. Is this route still in use? It seems to have the same purpose as /v1/reports/add